samples/sampler/stories/default/RangePicker.js

```diff
@@ -0,0 +1,30 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import RangePicker from '@enact/agate/RangePicker';
+
+const Config = mergeComponentMetadata('RangePicker', RangePicker);
+
+storiesOf('Agate', module)
+	.add(
+		'RangePicker',
+		() => (
+			<div style={{padding: '0 20%'}}>
+				<RangePicker
+					disabled={boolean('disabled', Config)}
+					onChange={action('onChange')}
+					min={number('min', Config, 0)}
+					max={number('max', Config, 20)}
+					step={number('step', Config, 1)}
+					defaultValue={number('defaultValue', Config, 5)}
```

Comment:
Even though changing the `defautValue` value in the Knobs, it didn't apply it to the RangePicker because the RangePicker already rendered. One possible solution is the one adding the `key={number('defaultValue', Config, 5)}` prop in the `RangePicker`. Then the RangePicker will render again when the `defaultValue` changes.

Reply:
Please write props alphabetically. So the `defaultValue` would be the first prop in the RangePicker.

Reply:
`defaultValue` shouldn't be used in samplers.   Use `value` and control it or don't modify the default value for already created controls.